pyiron_atomistics/lammps/interactive.py

```diff
@@ -114,12 +114,18 @@ def interactive_cells_setter(self, cell):
             warnings.warn('set_relative() is deprecated as of 2020-02-26. It is not guaranteed from pyiron_atomistics vers. 0.3')
```

Comment:
It's totally unrelated, but since this feature is badly deprecated, I guess we shouldn't forget to erase it after this PR.